storage/store.go

```diff
@@ -1280,11 +1283,16 @@ func raftEntryFormatter(data []byte) string {
 // updateStoreStatus updates the store's status proto.
 func (s *Store) updateStoreStatus() {
 	now := s.ctx.Clock.Now().WallTime
-	s.status.UpdatedAt = now
-	s.status.RangeCount = int32(len(s.ranges))
+	status := &proto.StoreStatus{
+		StoreID:    s.Ident.StoreID,
+		NodeID:     s.Ident.NodeID,
+		UpdatedAt:  now,
+		RangeCount: int32(len(s.ranges)),
+		Stats:      proto.MVCCStats(s.scanner.Stats().MVCC),
+	}
+	atomic.StorePointer(&s.status, unsafe.Pointer(status))
```

Comment:
I was tracking this down with Tobias this morning.  I kept running into a test race because of it was being written to via the scanner and the start() at the same time.  Most likely because of the very compressed timing of the canning iteration loop.

I'll be adding the ability to read it shortly, as I work on node status.